packages/adp-tooling/src/base/change-utils.ts

```diff
@@ -39,15 +39,15 @@ export function writeAnnotationChange(
     fs: Editor
 ): void {
     try {
-        const { fileName, answers } = data;
+        const { annotation } = data;
```

Comment:
The `data` parameter of this function is of type `AnnotationsData`. This means that the function requires  also`variant` to be provided, but it is never used. 
Here we see that only the `annotation` object is used from the `data` parameter.

Reply:
Changing the signature to:
```
export function writeAnnotationChange(
    projectPath: string,
    timestamp: number,
    { annotation }: AnnotationsData,
    change: ManifestChangeProperties,
    fs: Editor
): void {
```
should work (I think).

Reply:
I tried to change the type and use `Pick` to get only the `annotation` part, but as the properties are nested, it became quite ugly, so I followed Tobias' advice :) 

Fixed with: eb8529ef6a8aa2dc6f1d959c41e796f5e2d56d85

Reply:
But with this solution, the function still expects the whole object, including `variant`, to be provided.

Reply:
Very nice.

Reply:
Great, thanks!